packages/renderer/src/lib/image/BuildImageFromContainerfileCard.svelte

```diff
@@ -126,9 +126,9 @@ onMount(() => {
     </div>
     <div class="flex grow justify-end">
       {#if iconType === 'fontAwesome'}
-        <Fa class="text-[var(--pd-content-card-icon)] cursor-pointer" icon={icon} size="1.5x" />
+        <Fa class="text-[var(--pd-content-card-icon)] cursor-pointer" icon={icon as IconDefinition} size="1.5x" />
```

Comment:
issue: we can't cast from icon to IconDefinition

icon should have the type `IconDefinition | ....` rather than `unknown`